google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -341,8 +341,10 @@ def update
         # was used with {Bucket#create_file}, the `encryption_key` option must
         # be provided.
         #
-        # @param [String] path The path on the local file system to write the
-        #   data to. The path provided must be writable.
+        # @param [String, IO] path The path on the local file system to write the
```

Comment:
@blowmage I pulled this branch and locally get a Rubocop failure for line length here.